sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit bzr cmake-utils fdo-mime flag-o-matic gnome2-utils python-r1 vcs-snapshot wxwidgets versionator
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+
+SERIES=$(get_version_component_range 1-2)
+MY_PV=$(replace_version_separator 3 '-')
+
+LIB_REVISION="bd0ec6a20c7cc0556093f60eac723e4505947784"
+DOC_REVISION="0d0ed7b9de3afbba5d51fb6d7a25834000ed1099"
+I18N_REVISION="bf793d30d64f4e75ea7a109ea434c1a27926b158"
+
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${MY_PV}/+download/${PN}-${MY_PV}.tar.xz
+	!minimal? ( https://github.com/KiCad/${PN}-library/tarball/${LIB_REVISION} -> ${P}-library.tar.gz )
+	i18n? ( https://github.com/KiCad/${PN}-i18n/tarball/${I18N_REVISION} -> ${P}-i18n.tar.gz )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+boost debug doc examples github i18n minimal python webkit"
+LANGS="bg ca cs de el es fi fr hu it ja ko nl pl pt ru sk sl sv zh_CN"
+for lang in ${LANGS} ; do
+	IUSE="${IUSE} linguas_${lang}"
+done
+
+REQUIRED_USE="
+	github? ( webkit )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="x11-libs/wxGTK:${WX_GTK_VER}[X,opengl,webkit?]
+	python? (
+		dev-python/wxpython:${WX_GTK_VER}[opengl]
+		${PYTHON_DEPS}
+	)
+	boost? ( >=dev-libs/boost-1.54[context,threads,python?] )
+	dev-libs/openssl:0
+	media-libs/glew
+	media-libs/mesa
+	sys-libs/zlib
+	x11-libs/cairo"
+DEPEND="${CDEPEND}
+	>=dev-util/cmake-2.8.12.2
+	!boost? ( dev-vcs/bzr )
+	app-arch/xz-utils
+	doc? ( app-doc/doxygen )
+	i18n? ( >=sys-devel/gettext-0.18 )"
+RDEPEND="${CDEPEND}
+	sci-electronics/electronics-menu"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+src_prepare() {
```

Comment:
There are really too many seds in this src_prepare function. I bet no one really knows whether all of these even do something. sed also 'succeeds' when it doesn't find any match.